go.mod

```diff
@@ -178,7 +178,7 @@ require (
 	github.com/maratori/testpackage v1.0.1 // indirect
 	github.com/matoous/godox v0.0.0-20210227103229-6504466cf951 // indirect
 	github.com/mattn/go-colorable v0.1.8 // indirect
-	github.com/mattn/go-ieproxy v0.0.1 // indirect
+	github.com/mattn/go-iemultiplexer v0.0.1 // indirect
```

Comment:
```suggestion
	github.com/mattn/go-ieproxy v0.0.1 // indirect
```

Reply:
This is what currently makes your test fail. Should probably run `make` locally as well to get this cleaned up.